tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "noEmit": true,
     "allowJs": true,
     "checkJs": true,
-    "target": "ESNext"
+    "target": "ESNext",
```

Comment:
This makes it invalid JSON. Maybe TypeScript can handle that for its config files but then it's weird that the file is called `.json`. Let's get rid of this since I imagine this'll error in most editors. WDYT?

Reply:
Yeah I think TS is happy with this but agreed, this was an accidental by-product of some debugging anyway :)